demo/variable_outputs/run.py

```diff
@@ -0,0 +1,19 @@
+import gradio as gr
+
+max_textboxes = 10
+
+def variable_outputs(k):
+    k = int(k)
+    return [gr.Textbox.update(visible=True)]*k + [gr.Textbox.update(visible=False)]*(max_textboxes-k)
+
+with gr.Blocks() as demo:
+    s = gr.Slider(1, max_textboxes, step=1, label="How many textboxes to show:")
+    textboxes = []
+    for i in range(max_textboxes):
+        t = gr.Textbox(f"Textbox {i}")
```

Comment:
I think it would be better if only one textbox is showing on app load since that's the default value of the slider

Reply:
Sounds good, for simplicity of the demo, I'll just set the default value of the slider to be 10